app/templates/web-form.html

```diff
@@ -0,0 +1,206 @@
+                    'label': {
+                        'text': _('I am having technical difficulties')
+                    },
+                    'value': 'TECHNICAL'
+                },
+                {
+                    'id': 'form',
+                    'label': {
+                        'text': _('I need help with my census form')
+                    },
+                    'value': 'FORM'
+                },
+                {
+                    'id': 'complaint',
+                    'label': {
+                        'text': _('I have a complaint')
+                    },
+                    'value': 'COMPLAINT'
+                },
+                {
+                    'id': 'address',
+                    'label': {
+                        'text': _('I need help with my address')
+                    },
+                    'value': 'ADDRESS'
+                },
+                {
+                    'id': 'something_else',
+                    'label': {
+                        'text': _('Something else')
+                    },
+                    'value': 'OTHER'
+                }
+            ]
+        -%}
+
+{%- if 'error-country' in field_messages_dict -%}
+    {%- set error_country = {'id': 'error-country', 'text': _('Select an option')} -%}
+{%- endif -%}
+
+{%- if 'error-category' in field_messages_dict -%}
+    {%- set error_category = {'id': 'error-category', 'text': _('Select an option')} -%}
+{%- endif -%}
+
+{%- if 'error-description' in field_messages_dict -%}
+    {%- set error_description = {'id': 'error-description', 'text': _('Enter detail of your issue')} -%}
+{%- endif -%}
+
+{%- if 'error-name' in field_messages_dict -%}
+    {%- set error_name = {'id': 'error-name', 'text': _('Enter your name')} -%}
+{%- endif -%}
+
+{%- if 'error-email' in field_messages_dict -%}
+    {%- set error_email = {'id': 'error-email', 'text': _('Enter a valid email address')} -%}
+{%- endif -%}
+
+{%- block main -%}
+
+    {%- if messages_dict -%}
+        {%- include 'partials/messages.html' with context -%}
+    {%- endif -%}
+
+    <h1 class="u-fs-xxl u-mt-l">{{_('Web form')}}</h1>
+
+    {{
+        onsPanel({
+            'body': _('Do not include any personal information, for example, your access code'),
+            'type': 'warn'
+        })
+    }}
+
+    {{
+        onsRadios({
+            'legend': _('Select your country'),
+            'name': 'country',
+            'dontVisuallyHideLegend': true,
+            'value': form_value_country,
+            'radios': country_options,
+            'error': error_country
+        })
+    }}
+
+    {{
+        onsRadios({
+            'legend': _('What do you need help with?'),
+            'name': 'category',
+            'dontVisuallyHideLegend': true,
+            'value': form_value_category,
+            'radios': category_options,
+            'error': error_category
+        })
+    }}
+
+    {{
+        onsTextarea({
```

Comment:
This needs a value attribute? At present it loses any previous entered description if there is a problem with any other field:

       _onsTextarea({
            'id': 'description',
            'name': 'description',
            'classes': 'input--w-30',
            'label': {
                'text': _('Please provide more detail')
            },
            'value': form_value_description,_

Reply:
Added